test.js

```diff
@@ -80,12 +81,24 @@ async function waitForGruntServer() {
   return waitForExec('node', './node_modules/wait-on/bin/wait-on', 'http://127.0.0.1:9001');
 };
 
-async function cypressRun() {
-  if (argumentValues.testfiles) {
-    return asyncSpawn('node', './node_modules/cypress/bin/cypress', 'run', '--spec', `${argumentValues.testfiles}`, '--config', `{"fixturesFolder": "${argumentValues.outputdir}"}`);
-  }
+async function populateTestFiles() {
+  // accept the user-specified file(s)
+  if (argumentValues.testfiles) return;
 
-  return asyncSpawn('node', './node_modules/cypress/bin/cypress', 'run', '--config', `{"fixturesFolder": "${argumentValues.outputdir}"}`);
+  // otherwise, only include test files for plugins present in the course config
+  const config = JSON.parse(fs.readFileSync('./build/course/config.json'));
```

Comment:
```js
  const config = JSON.parse(fs.readFileSync('./build/course/config.json'));
```
Hard coded `./build/` ignores that outputdir is definable.

Reply:
Good catch, thanks Oliver